### DISCUSSION TITLE
How to set up social login locally?

### DISCUSSION THREAD

**Discussion by alveshelio** [nhost/nhost#469] [Q&A] [answered]
Couldn't find any documentation regarding this. Do I simply need to create an app for the local development and add the `client_id` and the `client_secret` to `config.yaml` and enable the social login I want? What should be the redirectURL for the local development app.  For example for LinkedIn for my production app I have https://my-backend-url.nhost.run/v1/auth/signin/provider/linkedin/callback what should be the url for my local development? Should it be http://localhost:1337/v1/auth/signin/provider/linkedin/callback?

**Comment by elitan** [comment, marked as answer]
> Do I simply need to create an app for the local development and add the client_id and the client_secret to config.yaml and enable the social login I want?

Yes 👍 

> Should it be http://localhost:1337/v1/auth/signin/provider/linkedin/callback?

Yes 👍

**Reply by elitan** [reply]
btw, here's an example of GitHub enabled locally:

![CleanShot 2022-04-26 at 21 24 24@2x](https://user-images.githubusercontent.com/331818/165376855-a0e4b297-2bd5-4c2a-9257-4a834426b6b0.png)

**Comment by alveshelio** [comment]
Hi Johan,

Thank you very much. This works
